src/hooks/HMSRoomProvider.tsx

```diff
@@ -75,14 +76,41 @@ export const HMSRoomProvider: React.FC<HMSRoomProviderProps> = ({
   );
 };
 
-/*
-UseHMSStore is a read only hook which can be passed a selector to read data.
-The hook can only be used in a component if HMSRoomProvider is present in its ancestors.
+/**
+ * `useHMSStore` is a read only hook which can be passed a selector to read data.
+ * The hook can only be used in a component if HMSRoomProvider is present in its ancestors.
  */
 export const useHMSStore = makeHMSStoreHook(HMSContext);
 
+/**
+ * `useHMSVanillaStore` is a read only hook which returns the vanilla HMSStore.
+ * Usage:
+ * ```
+ * const hmsStore = useHMSVanillaStore();
+ * const dominantSpeaker = hmsStore.getState(selectDominantSpeaker);
+ * ```
+ *
+ * Note: Refrain from using this in React components.
+ * This is used in rare cases where the store needs to be accessed outside a React component.
+ */
+export const useHMSVanillaStore = () => {
+  const HMSContextConsumer = useContext(HMSContext);
+  if (!HMSContextConsumer) {
+    const error =
+      'It seems like you forgot to add your component within a top level HMSRoomProvider, please refer' +
```

Comment:
let's link our docs here, also in the other error. also there is no space between 'refer' and 'to'(applies for the existing error too).

Reply:
let's reword to , please refer our integration docs(https://..) for the required steps in order to use this hook.